test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
I think the regular  validation attribute shoudln't work. The values are sent in a custom format with javascript.
https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/Views/ShoppingCart/Index.cshtml#L9-L13